build/projects/AI Test Toolkit/.AL-Go/settings.json

```diff
@@ -0,0 +1,17 @@
+{
+    "projectName": "AI Test Toolkit",
+    "appFolders":  [
+        "../../../src/Tools/AI Test Toolkit/App"
+    ],
+    "installOnlyReferencedApps": false,
```

Comment:
If there are no tests to be run, you can add these settings to avoid creating a container:

```
    "doNotRunTests": true,
    "useCompilerFolder": true,
    "doNotPublishApps": true,
```